arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -1678,6 +1681,16 @@ Result RestReplicationHandler::processRestoreDataBatch(transaction::Methods& trx
   bool const isUsersOnCoordinator = (ServerState::instance()->isCoordinator() &&
                                      collectionName == TRI_COL_NAME_USERS);
 
+  LogicalCollection* collection = trx.documentCollection(collectionName);
+  if (generateNewRevisionIds && !collection) {
+    return TRI_ERROR_ARANGO_DATA_SOURCE_NOT_FOUND;
+  }
+  PhysicalCollection* physical = collection->getPhysical();
+  if (!physical) {
+    return TRI_ERROR_ARANGO_DATA_SOURCE_NOT_FOUND;
+  }
+  char ridBuffer[11];
```

Comment:
std::byte?

Reply:
Method in which it is used calls for a `char*` for a buffer of length at least 11. This is the most straightforward satisfaction of those requirements. (It's also using it to generate actual string data, so it seems appropriate.)